challenges/objects-arrays.js

```diff
@@ -49,21 +69,34 @@ const graduates = [{"id":1,"first_name":"Cynde","university":"Missouri Southern
 /* Request 1: Create a new array called universities that contains all the universities in the graduates array.  
 
 Once you have the new array created, sort the universities alphabetically and log the result. */
+
 const universities = [];
-console.log(universities)
+  for (let i = 0; i < graduates.length; i++) {
+    universities.push(graduates[i].university)
+}
+console.log(universities.sort());
 
 /* Request 2: Create a new array called contactInfo that contains both first name and email of each student. 
 
 The resulting contact information should have a space between the first name and the email information like this: 
 Name email@example.com
 
 Log the result of your new array. */
+
 const contactInfo = [];
+  for (let i = 0; i < graduates.length; i++) {
+    contactInfo.push(graduates[i].first_name);
+    contactInfo.push(graduates[i].email);
+}
 console.log(contactInfo);
 
 
 /* Request 3: Find out how many universities have the string "Uni" included in their name. Create a new array called uni that contains them all. Log the result. */
+
 const uni = [];
+  for (let i = 0; i < graduates.length; i++) {
+  uni.push(graduates[i].university == 'Uni');
+}
 console.log(uni);
```

Comment:
Here you are looping through the array and checking whether or not the university equals "uni" since none of the universities are called Uni this is going to return false for every thing in the array.. Alternatively you can use the `includes` method which is a string method determines whether a string contains the given characters within it or not. This method returns true if the string contains the characters, otherwise, it returns false.